src/user.js

```diff
@@ -101,7 +101,6 @@ class User {
       }
     })
   }
-
```

Comment:
good, I always hated that sneaky hyphen

Reply:
Yes, love that you got rid of that meddling hyphen!